Sources/Plot/API/HTMLAttributes.swift

```diff
@@ -297,6 +297,22 @@ public extension Attribute where Context == HTML.InputContext {
     }
 }
 
+public extension Attribute where Context == HTML.ButtonContext {
```

Comment:
Is this needed, given that a button can only contain `Node` instances, not `Attribute` ones?

Reply:
Good catch. Fixed in 98c2a48af319acfdd866b1a7b6fc88d970e3d688.